calculator/src/main/resources/application.yml

```diff
@@ -1,10 +1,28 @@
 spring:
   application:
     name: calculator
+  datasource:
+    url: jdbc:postgresql://localhost:5432/innowise
+    username: postgres
+    password: 123414
+  jpa:
+    properties:
+      hibernate:
+        dialect: org.hibernate.dialect.PostgreSQL9Dialect
+    hibernate:
+      ddl-auto: create-drop
   cloud:
     consul:
       host: localhost
       port: 8500
       discovery:
         health-check-interval: 20s
-        health-check-timeout: 6s
\ No newline at end of file
+        health-check-timeout: 6s
+      config:
+        enabled: false
+  activemq:
+    broker-url: tcp://localhost:61616
+    user: admin
+    password: admin
```

Comment:
You should never commit any credentials to your property files. That's a common rule with no exception. Either you can use a consul config for this or inject those creds via environment variables.